docs/databricks.koala.rst

```diff
@@ -0,0 +1,51 @@
+databricks.koala package
+==========================
```

Comment:
As mentioned, we do not need to document the koala package, but only:
 - SparkSession: https://github.com/databricks/spark-pandas/blob/master/databricks/koala/utils.py#L63
 - DataFrame: https://github.com/databricks/spark-pandas/blob/master/databricks/koala/utils.py#L52
 - Column: https://github.com/databricks/spark-pandas/blob/master/databricks/koala/utils.py#L53

So `pyspark.sql.dataframe` etc. should be the only 3 required modules.